docs/fleet/index.md

```diff
@@ -17,32 +18,53 @@ At first glance, you can see the following data in the table:
 * *Infra App*: the Infra App which is receiving the data from the Sematext Agent. Clicking on the App name will navigate you to the overview of the corresponding App in the [Infrastructure](https://sematext.com/docs/monitoring/infrastructure/) section of Sematext Cloud.
 
 # Agent Information Panel
+
 ## Info
 The Info panel contains information about the host system and the installed Agents. 
 
 If the Agents are installed in a Kubernetes environment, you will see information about the Kubernetes environment, like Namespace and Node. If, however, the Agents are installed in a baremetal environment, this panel will show information related to the Agent process, like the Pid. 
 
 You can also find a link to the Infra App the Agent is associated with.
+
 ![Agent Info](../images/fleet/fleet-agent-info.png)
+
 ## Log Agents
 
 This panel contains a list of the Log Shippers installed on the host. If you click on one, it will take you to the Log Shipper panel, where you will find information about the Log Shipper version and type, the files and containers it's collecting logs from, as well as any errors. You can also see diagnostics data and report a problem from the Diagnostics tab.
+
 ![Log Agents](../images/fleet/fleet-log-agents.png)
+
+The following status values are currently available for Log Shippers:
```

Comment:
@cpipilas The next logical step might be to suggest how to troubleshoot / fix log shippers with status Dead or Init Failure, maybe also Stopped and Not Configured.

Reply:
Updated